app/main.py

```diff
@@ -14,4 +14,5 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    list = [i for i in phrase.lower() if letter.lower() == i]
```

Comment:
Good solution but please, use descriptive and correct variable names (change the variable name `i`).